### ISSUE TITLE
Model Project feedback

### ISSUE THREAD

**Issue by Holger-Harmsen** [NumEconCopenhagen/projects-2020-crystal-ebba-eva-vanessa#10]
Cool parts:
- Really cool using sympy and having Python print out beautiful equations; I had used more manual and less sophisticated ways of solving the equations. 
- Cool that you check your calculations using two different methods
- Nice plot!
- Nice walkthrough of your methodology, introduction and conclusion. It is clear in what direction you’re heading.

Unclear parts:
- It is cool that you extend the model - however, you do the exact same steps again; it would be cool with some different analysis the second time around;
- Could be nice with a simulation over time with stochastic shock maybe - or just showing convergence back to SS after a shock.

In conclusion:
- Really nice project - beautiful equation solving - but could be nice with time simulations, e.g. with shocks.
